src/Sylius/Bundle/ApiBundle/CommandHandler/SendContactRequestHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler;
+
+use Sylius\Bundle\ApiBundle\Command\SendContactRequest;
+use Sylius\Bundle\CoreBundle\Mailer\Emails;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Mailer\Sender\SenderInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Webmozart\Assert\Assert;
+
+/** experimental */
+final class SendContactRequestHandler implements MessageHandlerInterface
```

Comment:
We should write a functional test for this class, to ensure that it may trigger email without shop bundle. Take a look at `\Sylius\Bundle\ApiBundle\Tests\CommandHandler\SendResetPasswordEmailHandlerTest` to know how to do it